tests/security/apparmor/aa_easyprof.pm

```diff
@@ -0,0 +1,84 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: AppArmor profile generation made easy testing
+# Maintainer: Wes <whdu@suse.com>
+# Tags: poo#36895, tc#1621144
+
+use strict;
+use base "apparmortest";
+use testapi;
+use utils;
+use version_utils qw(is_sle is_leap);
+
+sub run {
+    my ($self) = @_;
+
+    my $aa_easyprof_jsonopt;
```

Comment:
I think it is too much . There is golden rule- " if ALL your variables has some static prefix drop it. ".

Reply:
Too much for what? Sorry I don't understand. Would you please explain?

Reply:
I was trying to say that aa_easyprof is filename so having all variables prefixed with this name does not bring any value so should be removed. I used bad wording for this , sorry .

Reply:
Good suggestion, I will make it shorter, thanks